src/host/lua_auxlib.c

```diff
@@ -103,6 +110,9 @@ LUALIB_API int luaL_loadfilex (lua_State* L, const char* filename, const char* m
 	/* Either way I should have ended up with the file name followed by the
 	 * script chunk on the stack. Turn these into a closure that will call my
 	 * wrapper below when the loaded script needs to be executed. */
+
+	assert(lua_gettop(L) == bottom + 2);
```

Comment:
Do we want an assertion failure here, or should we report the error more gracefully somehow?